applications/AdjointFluidApplication/custom_schemes/adjoint_bossak_scheme.h

```diff
@@ -72,11 +73,16 @@ class AdjointBossakScheme : public Scheme<TSparseSpace, TDenseSpace>
         Parameters default_params(R"(
```

Comment:
Not in the scope of this PR but `Kratos::Parameters` should be passed by value according to Riccardo